scripts/publish.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
```

Comment:
all scripts are in js or bash, is introducing python here really necessary?
If you don't want to deal with cmd parsing in bash you could always pass everything by env instead.